libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/TxBodySpec.hs

```diff
@@ -276,3 +284,27 @@ go2 = do
   putStrLn (show (pcTxBodyWithUTxO utxo txbody))
   print (consumedTxBody txbody (certsPParams certsEnv) certState utxo)
   print (producedTxBody txbody (certsPParams certsEnv) certState)
+
+go :: IO ()
+go = do
```

Comment:
Please, don't create top level bindings with `go` name. It is very often used for local bindings.

As usual I will suggest removing the whole function altogether, since it is dead code

Reply:
So I changed the name, to testBody. This whole file is the experiment of using Witnesses, so having a easy way to try out changes is necessary. When things are complete, we can get rid of it.